### DISCUSSION TITLE
fstab mount on ubuntu 18.04

### DISCUSSION THREAD

**Discussion by dragon2611** [moosefs/moosefs#338] [Q&A]
Not sure if it's worth adding to the documentation but I found I had to add the following to the end of the fstab command otherwise the system would not boot cleanly.

`,x-systemd.mount-timeout=30,_netdev`

The line in fstab now looks like

`mfsmount /mnt/mymnt fuse mfssubfolder=mydir,allow_other,x-systemd.mount-timeout=30,_netdev`

I think it was trying to mount mooseFS before it had finished connecting to the network, which of course is almost always going to end badly.

**Comment by BradFD** [comment]
Thank you. This worked. I was having trouble getting my first client mount working on my first moosefs cluster.

**Comment by pkonopelko** [comment]
You can also use `systemd` mount unit file as described [here](https://github.com/moosefs/moosefs/issues/123#issuecomment-397565138).
Piotr / MooseFS Team
